sycl/test/check_device_code/ap_fixed.cpp

```diff
@@ -8,103 +8,59 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "CL/__spirv/spirv_ops.hpp"
+#include <sycl/sycl.hpp>
```

Comment:
Why do we need the whole `sycl/sycl.hpp` here? Especially if we don't use kernel submission API anymore

Reply:
My bad! My editor was complaining about missing certain SYCL features, so I had assumed that I needed sycl.hpp.

This is fixed now!